app/models/spotlight/reindexing_log_entry.rb

```diff
@@ -0,0 +1,42 @@
+module Spotlight
+  ##
+  # a log entry representing an attempt to reindex some number of records in an exhibit
+  class ReindexingLogEntry < ActiveRecord::Base
+    enum job_status: { unstarted: 0, in_progress: 1, succeeded: 2, failed: 3 }
+
+    belongs_to :exhibit, class_name: 'Spotlight::Exhibit'
+    belongs_to :user, class_name: '::User'
+
+    # null start times sort to the top, to more easily surface pending reindexing
+    default_scope { order('start_time IS NOT NULL, start_time DESC') }
+    scope :recent, -> { limit(5) }
+
+    def duration
+      end_time - start_time if end_time
+    end
+
+    def reindexing_started!
+      self.start_time = Time.zone.now
+      self.job_status = :in_progress
+      save
+    rescue
+      Rails.logger.error "unexpected error updating log entry to :in_progress from #{caller}"
```

Comment:
i think this was more of a vague feeling that reindexing a bunch of things asynchronously has the potential for errors... but thinking about it more, i guess something like a network error connecting to solr seems like the most likely possible problem to me?

also, it seems like it'd be a good idea to log the actual error.  not sure if that was part of what you were getting at, but seems like an oversight, now that i'm thinking about that line again.

would you also prefer a more realistic example error in the test?  or a comment in the test as to what sorts of things we might expect?